akka-http-core/src/main/scala/akka/http/impl/engine/parsing/BodyPartParser.scala

```diff
@@ -289,5 +292,6 @@ private[http] object BodyPartParser {
     maxHeaderCount = 64,
```

Comment:
I don't think there is anything missing here.
It's simply another settings class that needs to be introduced, along with the proper defaults and docs in `reference.conf`. Basically just "work"... :)